xml/System.Threading.Tasks/Parallel.xml

```diff
@@ -2478,12 +2487,21 @@
         <Parameter Name="body" Type="System.Func&lt;TSource,System.Threading.CancellationToken,System.Threading.Tasks.ValueTask&gt;" Index="1" FrameworkAlternate="net-6.0" />
       </Parameters>
       <Docs>
-        <typeparam name="TSource">To be added.</typeparam>
-        <param name="source">To be added.</param>
-        <param name="body">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <typeparam name="TSource">The type of the data in the source.</typeparam>
+        <param name="source">An enumerable data source.</param>
+        <param name="body">An asynchronous delegate that is invoked once per element in the data source.</param>
+        <summary>Executes a for each operation on an <see cref="T:System.Collections.Generic.IEnumerable`1" /> in which iterations may run in parallel.</summary>
+        <returns>A task that represents the entire for each operation.</returns>
```

Comment:
Word missing.

Reply:
It's correct. Maybe it needs a dash between `for-each`?

Reply:
```suggestion
        <returns>A task that represents the entire <c>for-each</c> operation.</returns>
```